runtime-config-linux.md

```diff
@@ -17,7 +17,7 @@ The following parameters can be specified to setup namespaces:
     * **`uts`** the container will be able to have its own hostname and domain name
     * **`user`** the container will be able to remap user and group IDs from the host to local users and groups within the container
 
-* **`path`** *(string, optional)* - path to namespace file
+* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
```

Comment:
:disappointed: 

It is the namespace of the runtime (not host) as you can nest things so this is not exactly correct to say.